<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nmicCommand</class>
 <widget class="QDockWidget" name="nmicCommand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>237</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>237</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Commands</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QPushButton" name="stim_transfer">
      <property name="text">
       <string>STIM_TRANSFER</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="stim_start">
      <property name="text">
       <string>STIM_START</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="reset">
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="hv_load">
      <property name="text">
       <string>HV_LOAD</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="imp_start">
      <property name="text">
       <string>IMP_START</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="clear_err">
      <property name="text">
       <string>CLEAR_ERR</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="stim_reset">
      <property name="text">
       <string>STIM_RESET</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
